src/components/Organizations/Manage/index.js

```diff
@@ -123,22 +124,32 @@ const Manage = ({ users = [], organization, organizationId, organizationName, re
       },
     },
     {
-      width: '10%',
+      width: '15%',
       key: 'actions',
       render: ({ ...user }) => {
+        const linkData = getLinkData(user.email, organizationId, organizationName);
         return (
           <TableActions>
-            <span
-              className="link"
-              onClick={() => {
-                setSelectedUser(user?.id);
-                setEditModalOpen(true);
-                setSelectedUserOwner(user.owner);
-              }}
-            >
-              <EditOutlined className="edit" />
-            </span>
+            <Tooltip overlayClassName="orgTooltip" placement="bottom" title="View user">
+              <>
+                <Link href={linkData.urlObject} as={linkData.asPath}>
```

Comment:
The view user links here fail if the administator isn't in any groups in the organization. A user can be an administator but not a user/in any groups in the org. Not sure if these view user links are necessary on this page?

Reply:
Good point, the `userByEmailAndOrganization` query returns null on the user page for those kind of users.
What if we have a a different view on this page if that happens? 
We could also do a check in the background - if the query returns null we disable the links for those specific users? 
@shreddedbacon @tobybellwood

Reply:
That is a lot of queries to perform when rendering the page potentially, a lot of these user queries are not very cached at the moment. So performing them often *could* see some performance impacts over time.

If you can handle the `null` user experience with some sort of message that says the user is not in any groups, then that would be fine.

Alternatively, not linking the administrators of an organization to the users that are contained within the groups is also a pretty good idea.

Reply:
I'm worried that there is going to be too much circular references in the organizations UI component that will get people clicking around and landing in all sorts of spots.

Like, the administrators of the organization are just that, and linking the user OUT of the management view into a view where they can then manage the users who are contained within the groups of the organization, which then leads them into all other parts of the management of organizations.

If I know I need to manage the organizations administrators, simplying going to manage to add, edit, or remove a user as an admin is all I should be able to do there, IMO